src/MongoDB.Analyzer/Core/Builders/BuilderExpressionProcessor.cs

```diff
@@ -143,11 +143,32 @@ private static IEnumerable<SyntaxNode> GetBuildersDefinitionNodes(SemanticModel
             {
                 nodesProcessed.Add(node);
 
-                // Skip MongoDB.Driver.Builders<T> nodes
+                var isBuildersContainer = false;
+                var containsAlias = false;
+
+                //Check if Member Expression is Builders Container
```

Comment:
Please add space after `//` in all comments

Reply:
Done